e2e/credentials-server.e2e.spec.ts

```diff
@@ -0,0 +1,177 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * This file includes end-to-end testing for the HTTP control interface
+ */
+
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import { spawnSync } from 'child_process';
+
+import { expect, test } from '@playwright/test';
+import { BrowserContext, ElectronApplication, Page, _electron } from 'playwright';
+
+import fetch from 'node-fetch';
+import { createDefaultSettings, playwrightReportAssets } from './utils/TestUtils';
+import paths from '@/utils/paths';
+import { ServerState } from '@/main/commandServer/httpCommandServer';
+import { spawnFile } from '@/utils/childProcess';
+import { findHomeDir } from '@/config/findHomeDir';
+
+function haveDockerCredentialAssistant(): boolean {
```

Comment:
These are credential stores, not credential assistants.

Reply:
Actually, they are credential helpers. 😛 https://github.com/docker/docker-credential-helpers

Reply:
Settling on the name `httpCredentialHelperServer`. Should be read from right to left, but this is a better name than `credentialHelperHttpServer`.

Reply:
Yes, missed that.